api/src/main/java/org/openmrs/OrderGroup.java

```diff
@@ -0,0 +1,155 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs;
+
+import java.io.Serializable;
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+/**
+ * Transcational Data for the OrderSet {@link org.openmrs.OrderSet}
+ * Contains the OrderSet to be ordered for a particular patient, in a specific encounter
+ */
+public class OrderGroup extends BaseOpenmrsData implements Serializable {
+	
+	public static final long serialVersionUID = 72232L;
+	
+	private Integer orderGroupId;
+	
+	private Patient patient;
```

Comment:
I know but it can it is not necessary in my opinion and it provides room for error and now you have ensure that encounter.patient is equal to orderGroup.patient